### PR TITLE
wdio-cli: fix local runner in config

### PR DESCRIPTION
**Pull request by mgrybyk**
## Proposed changes

Use a proper name for wdio local runner #6226 

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by mgrybyk** [comment]
tests are passing locally for me if I rollback `rgb2hex` to `0.2.3`

File: packages/wdio-cli/tests/commands/config.test.ts [hunk 1]
```diff
@@ -98,16 +97,14 @@ test('should throw an error if something goes wrong', async () => {
     }
 })
 
-test('prints TypeScript setup message', async () => {
-    (convertPackageHashToObject as jest.Mock).mockImplementation((input) => input)
-    ;(inquirer.prompt as any as jest.Mock).mockReturnValue(Promise.resolve({
+test.only('prints TypeScript setup message', async () => {
```
Comment (christian-bromann) [inline packages/wdio-cli/tests/commands/config.test.ts:100]:
```suggestion
test('prints TypeScript setup message', async () => {
```

**Review by christian-bromann — APPROVED** [review]
One comment but LGTM 👍 
Thanks!

File: packages/wdio-cli/tests/commands/config.test.ts (hunk 1 above)
Comment (mgrybyk) [reply packages/wdio-cli/tests/commands/config.test.ts]: Oh, missed that, nice catch!

**Pull request merged**